### PR TITLE
CB-11112 Added Appium documentation

### PR DESCRIPTION
**Pull request by alsorokin**
https://issues.apache.org/jira/browse/CB-11112

### CODE REVIEW

File: APPIUM.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ ``` shell
+ > brew update
+ > brew install ios-webkit-debug-proxy
+ ```
+
+More info on installing `ios-webkit-debug-proxy` can be found in [Appium docs][appium_docs].
+
+If running on simulator, no need to launch it manually - Appium will do it automatically.
+
+Command sample to run tests on the "iPhone 6" simulator with iOS 8.4:
```
Comment (rakatyal) [inline APPIUM.md]: Should we include an example to run it on a device for android/ios?

File: APPIUM.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+``` shell
+ > cordova create test-app
+ > cd test-app
+ > cordova platform add <android or ios>
+ > cordova plugin add cordova-plugin-camera cordova-plugin-contacts
+```
+
+### Running on Android
+
+Cordova Appium tests for Android can be ran on either real device or an emulator. If you're running tests on the emulator you will need to launch it manually before the test run. Please also note that Appium tests are written with Android API 19 (KitKat) and 21 (Lollipop) in mind. They're not yet tested on higher versions of Android OS.
```
Comment (rakatyal) [inline APPIUM.md]: Are there any OS/software requirements to run Appium tests? Like min OSX version/Xcode version or something?

**Comment by rakatyal** [comment]
Minor comments. Otherwise LGTM.

File: APPIUM.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
```
Comment (riknoll) [inline APPIUM.md]: Grammar nitpick:
Cordova Appium tests for Android can be *run* on either *a* real device or an emulator.

*They* have not yet *been* tested on higher versions of Android

File: APPIUM.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+This document describes a method to run Cordova Appium tests without full-blown CI setup with [Camera][plugin_camera] and [Contacts][plugin_contacts] plugins taken as an example.
+Appium tests can be ran on Android and iOS.
+
+## Setup
+
+Clone medic repository by running
+
+``` shell
+ > git clone https://github.com/apache/cordova-medic
```
Comment (riknoll) [inline APPIUM.md]: You also need to run `npm install` in the repo

**Comment by riknoll** [comment]
Very helpful guide! I was able to get the tests running on my machine (for Android, didn't try iOS). One thing to note is that it seems like the tests are very dependent on the apps installed on the device running them and it might be helpful to document that. For example, the tests for camera that use PHOTOLIBRARY as a source do not work with the gallery app that comes with the VS Android emulator (they just time out).

File: APPIUM.md (hunk 3 above)
Comment (alsorokin) [inline APPIUM.md]: Thanks a lot for this. I clearly need to practice English more.

**Comment by alsorokin** [comment]
@riknoll @rakatyal Addressed review notes, please see https://github.com/apache/cordova-medic/pull/93/commits/0e7ff36d9a53d72e361eb86574a224f904e535e6

File: APPIUM.md [hunk 5]
```diff
@@ -58,7 +71,17 @@ Command sample to run tests on the "iPhone 6" simulator with iOS 8.4:
...
+
+``` shell
+ > node cordova-medic/medic/medic.js appium --app test-app --platform ios --deviceName "iPad 2" --platformVersion 8.1 --plugins "cordova-plugin-camera cordova-plugin-contacts" --device --udid <device UDID>
+```
+
 [appium_docs]:     https://github.com/appium/appium/blob/master/docs/en/advanced-concepts/ios-webkit-debug-proxy.md
 [brew]:            http://brew.sh/
 [plugin_camera]:   https://github.com/apache/cordova-plugin-camera
 [plugin_contacts]: https://github.com/apache/cordova-plugin-contacts
+[appium_docs]:     http://appium.io/getting-started.html
```
Comment (rakatyal) [inline APPIUM.md]: There is already [appium_docs] defined so consider renaming it. Also you may directly point it to 'http://appium.io/getting-started.html#requirements'.

Comment (alsorokin) [inline APPIUM.md]: Nice catch. Fixed

**Comment by rakatyal** [comment]
LGTM.

**Comment by riknoll** [comment]
LGTM

**Comment by alsorokin** [comment]
Thanks guys. Merging

**Pull request merged**